flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -682,27 +729,27 @@
 	 */
 	public static final int DEFAULT_FS_STREAM_OPENING_TIMEOUT = 0;
 
-	// ------------------------ YARN Configuration ------------------------
+
+	// ------ Common Resource Framework Configuration (YARN & Mesos) ------
 
 	/**
-	 * Minimum amount of Heap memory to subtract from the requested TaskManager size.
-	 * We came up with these values experimentally.
-	 * Flink fails when the cutoff is set only to 500 mb.
+	 * Minimum amount of memory to subtract from the process memory to get the TaskManager
+	 * heap size. We came up with these values experimentally.
 	 */
-	public static final int DEFAULT_YARN_MIN_HEAP_CUTOFF = 600;
+	public static final int DEFAULT_CONTAINERED_MIN_HEAP_CUTOFF = 600;
 
 	/**
-	 * Relative amount of memory to subtract from the requested memory.
+	 * Relative amount of memory to subtract from Java process memory to get the TaskManager
+	 * heap size
 	 */
-	public static final float DEFAULT_YARN_HEAP_CUTOFF_RATIO = 0.25f;
+	public static final float DEFAULT_CONTAINERED_HEAP_CUTOFF_RATIO = 0.25f;
```

Comment:
You can not rename variables in the ConfigConstants because the class is publicly visible.